### PR TITLE
Try to get tagged post to publish on wpcalypso

### PR DESCRIPTION
**Pull request by alisterscott**
Work in Progress

Issues with the editor and saving tags raised here: https://github.com/Automattic/wp-calypso/issues/28974

### CODE REVIEW

File: lib/gutenberg/gutenberg-editor-sidebar-component.js [hunk 1]
```diff
@@ -148,6 +148,12 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 		return await tagEntryElement.sendKeys( Key.ENTER );
 	}
 
+	async tagEventuallyDisplayed( tag ) {
+		//const selector = await driverHelper.getElementByText( this.driver, 'span', tag );
```
Comment (alisterscott) [inline lib/gutenberg/gutenberg-editor-sidebar-component.js]: @Stojdza any ideas why this doesn't work? I've tried it locally and it doesn't work

File: specs/wp-calypso-gutenberg-post-editor-spec.js [hunk 2]
```diff
@@ -152,6 +158,16 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			);
 		} );
 
+		step( 'Can see the post tag in preview', async function() {
```
Comment (alisterscott) [inline specs/wp-calypso-gutenberg-post-editor-spec.js]: This should make this fail faster if it isn't saved in the post

File: specs/wp-gutenberg-post-editor-spec.js [hunk 3]
```diff
@@ -152,6 +158,16 @@ describe( `[${ host }] Gutenberg Editor: Posts (${ screenSize })`, function() {
 			);
 		} );
 
+		step( 'Can see the post tag in preview', async function() {
```
Comment (alisterscott) [inline specs/wp-gutenberg-post-editor-spec.js]: This should make this fail faster if it isn't saved in the post

**Comment by alisterscott** [comment]
This still isn't consistently passing on CI

File: lib/gutenberg/gutenberg-editor-sidebar-component.js (hunk 1 above)
Comment (Stojdza) [reply lib/gutenberg/gutenberg-editor-sidebar-component.js]: Yeah, selector should be `By.css( 'span' )` not just `'span'`. 

`getElementByText()` probably should be more clear about selectors - should we send, for example, `<className>` or `By.css( '.<className>' )`

**Comment by Stojdza** [comment]
@alisterscott I disabled this same test for Gutenberg Editor also, it was failing on master https://github.com/Automattic/wp-e2e-tests/pull/1680

Can we disable this step where we are checking tag, and enable tests again (until https://github.com/Automattic/wp-calypso/issues/28974 is solved)?

File: lib/gutenberg/gutenberg-editor-sidebar-component.js (hunk 1 above)
Comment (alisterscott) [reply lib/gutenberg/gutenberg-editor-sidebar-component.js]: Yeah 🤦‍♂️

**Comment by alisterscott** [comment]
> Can we disable this step where we are checking tag, and enable tests again (until Automattic/wp-calypso#28974 is solved)?

Should I disable checking it is added to the sidebar as well? Or just the front end?

**Comment by Stojdza** [comment]
> Should I disable checking it is added to the sidebar as well? Or just the front end?

I would say just front end, since sidebar check is passing. I'll disable it and check results.

**Comment by Stojdza** [comment]
I'll merge this now and see how's it going on master

**Pull request merged**